mobile.js

```diff
@@ -0,0 +1,27 @@
+const eventMobile = document.querySelector('.ic_menu');
+
+const target1 = document.querySelector('.fa-sharp');
+const target2 = document.querySelector('.fa-xmark');
```

Comment:
- [ ] After one of the `project`, `portfolio` or `about` links in the menu display is clicked, the menu display disappears as expected but the `X` close icon replaces the hamburger icon as shown in **_figure 1_**  below 👇🏾 . And if I click on the close Icon which should actually be the hamburger icon, the menu display appears as expected but the hamburger icon replaces the close icon as shown in **_figure 2_** below 👇🏾 

#### Figure 1
![image](https://user-images.githubusercontent.com/84156714/223491908-cf6881fe-f2a5-4010-81bd-70f861cb76e0.png)

#### Figure 2
![image](https://user-images.githubusercontent.com/84156714/223492542-014e4007-f11c-442c-8012-b8b338ba3033.png)


**Every other thing seems to work as expected.**